README.md

```diff
@@ -1,2 +1,109 @@
-# unione-php
-UniOne Email API PHP Library
+# UniOne PHP client
+
+[![Latest Stable Version](http://poser.pugx.org/unione/unione-php/v)](https://packagist.org/packages/unione/unione-php)
+[![Total Downloads](http://poser.pugx.org/unione/unione-php/downloads)](https://packagist.org/packages/unione/unione-php)
+
+This SDK contains methods for easily interacting with the UniOne API: https://docs.unione.io/en/web-api-ref#web-api
+
+## Installation
+
+Use Composer to install the package:
+
+```bash
+composer require unione/unione-php
+```
+
+## Usage
+
+### Send an email:
+API [documentation](https://docs.unione.io/en/web-api-ref#email).\
+Add data fields to email [documentation](https://docs.unione.io/en/simple-template-engine).
+
+```php
+  use Unione\UniOneClient;
+
+  $recipients = ['example@example.com', 'another@example.com'];
+
+  $body = [
+    "html" => "<b>Test mail, {{to_name}}</b>",
+    "plaintext" => "Hello, {{to_name}}",
+    "amp" => "<!doctype html><html amp4email><head> <meta charset=\"utf-8\"><script async src=\"https://cdn.ampproject.org/v0.js\"></script> <style amp4email-boilerplate>body[visibility:hidden]</style></head><body> Hello, AMP4EMAIL world.</body></html>"
+  ];
+
+  $mail = new Email($recipients, $body);
+  $mail->setFromEmail('test@unione.io');
+
+  // https://us1.unione.io/en/transactional/api/v1/ - UniOne USA & Canada Instance
+  // https://eu1.unione.io/en/transactional/api/v1/ - UniOne European Instance
+  // First, instantiate the client with your API credentials.
+  $client = new UniOneClient('https://eu1.unione.io/en/transactional/api/v1/', 'api-key');
+
+  // Now, compose and send your email.
+  $response = $client->emails()->send($mail);
+```
+
+### Send a subscribe email:
+API [documentation](https://docs.unione.io/en/web-api-ref#email-subscribe).
+
+```php
+  $params = [
+    "from_email" => "test@unione.io",
+    "from_name" => "string",
+    "to_email" => "test@example.com"
+  ];
+
+  // Now, send your subscribe email.
+  $response = $client->emails()->subscribe($params);
+```
+
+### Set a template:
```

Comment:
Why we don't add examples with other Template methods?

Reply:
I removed them when cleaned Documentation from not necessary methods description. Now I returned them.